x-pack/plugins/fleet/common/types/rest_spec/app.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 export interface CheckPermissionsResponse {
-  error?: 'MISSING_SECURITY' | 'MISSING_SUPERUSER_ROLE';
+  error?: 'MISSING_SECURITY' | 'MISSING_PRIVILEGES';
```

Comment:
Do we need to update the OpenAPI spec for this change?

Reply:
I don't think we have that endpoint in our openApi specs but I can add it

Reply:
If we don't have it, even better. This should be considered private I think.